hugo.toml

```diff
@@ -18,11 +18,23 @@ title = 'MaxMind'
     tabWidth = 4
 
 [menus]
-  [[menus.main]]
+  [[menus.minfraud]]
     name = 'minFraud'
     pageRef = '/minfraud'
     weight = 10
-  [[menus.main]]
+  [[menus.geoip]]
     name = 'GeoIP'
     pageRef = '/geoip'
-    weight = 20
+    weight = 10
```

Comment:
Why the change in `20` to `10`? I thought the `weight` is used to indicate the ordering of the sections in the menu, but maybe I'm wrong.
> weight
(int) A non-zero integer indicating the entry’s position relative the root of the menu, or to its parent for a child entry. Lighter entries float to the top, while heavier entries sink to the bottom.

https://gohugo.io/content-management/menus/#define-in-site-configuration

Reply:
This is now the root of the new `geoip` section

Reply:
Yeah I guess I was confused at first because I didn't see the same weight for any of the examples.